examples/with-msw/pages/_app.js

```diff
@@ -1,7 +1,13 @@
-if (process.env.NEXT_PUBLIC_API_MOCKING === 'enabled') {
+// Enable API mocking in all environments, this is only for the sake of the example.
+// In a real app you should remove this code and uncomment the code below.
+if (true) {
   require('../mocks')
 }
 
+// if (process.env.NODE_ENV !== 'production') {
+//   require('../mocks')
+// }
```

Comment:
This should be the default 👍

Reply:
I agree! But since there is no error handling in the `getStaticProps`, it will cause a `FetchError` in production that results in a 500 Internal Server Error response. Should I add error handling to the `getStaticProps` function to make it work in production for the sake of the example?

Reply:
Agree. Next examples serve as starters, so we can expect people using this example to build actual production projects. Shipping API mocking as always enabled would require a necessary step to configure the mocking, while we know a sensible pre-configuration.

Reply:
Should I let the error be? I've got another suggestion: I could create a Next.js API route that is mocked by MSW, and then fetch that API from React. That would mean the example works in both development and production. And then we can still provide a sensible default configuration.

Reply:
@jensmeindertsma Add error handling to `getServerSideProps` with comments about why it's not going to work on production (there's no db and only the mock works)

Reply:
Just pushed some more changes that implement error handling inside `getServerSideProps`. Also added notes to both the requests regarding the FetchError that occurs in production.